stock_quant_strap_product/views/stock_quant_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_stock_quant_tree" model="ir.ui.view">
+        <field name="name">view.stock.quant.tree</field>
+        <field name="model">stock.quant</field>
+        <field name="inherit_id" ref="stock_view_adjust_oaw.view_stock_quant_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='image_medium']" position="after">
+                <field name="strap_product_id" domain="[('name','ilike', 'strap')]" />
```

Comment:
Wouldn't it be better to have a separate indicator (i.e. a boolean field) in product? Relying on the product name is more prone to mistakes and sort of limits usability than doing otherwise.

Reply:
Yes, you are right, I have added a field to product.template to indicate whether it is a strap product.